microsetta_interface/implementation.py

```diff
@@ -2480,12 +2522,22 @@ def admin_barcode_search_query_qiita(body):
 
 
 def get_ajax_check_kit_valid(kit_name):
+    if re.match(
+            r"[dD][mM][kK][234689ACDEFHJKMNPRTVWXYacdefhjkmnprtvwxy]{6}",
```

Comment:
why this and not `re.match(r"DMK[234689ACDEFHJKMNPRTVWXY]{6}", kit_name.upper())`? I think its the same in the end but maybe im missing something. If it is, benefit is easier to read regex imo...

Reply:
Good call, adjusting.